imports/ui/components/users-search-list.jsx

```diff
@@ -57,7 +58,7 @@ class UsersSearchList extends Component {
                         </div>
                         <div className='ml2 flex-grow overflow-hidden'>
                           <div className={'f5 ellipsis ' + (user.pending ? 'i silver' : 'bondi-blue')}>
-                            {user.name || user.login}
+                            { username }
```

Comment:
The desired functionality is that when a user has a `name` (the user's full name), and otherwise we use the email address to show the left part of it as its placeholder name.
This code will never show name for user even if it is defined... it will just show the left part of the email...